src/main/java/com/billennium/bcvsaver/entity/TechnicalSkill.java

```diff
@@ -0,0 +1,24 @@
+package com.billennium.bcvsaver.entity;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Data
+public class TechnicalSkill {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToMany(mappedBy = "technicalSkills")
+    private Set<Technology> technologies = new HashSet<>();
+
+    // it can be values like 1.5
+    private Float expInYears;
+
+    // it will be enum 1-5
+    private int lvlOfKnowledge;
```

Comment:
Where is `@Column` with db column name ?
If you use int (primitive type) column should be set as not nullable.